<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cubedialog</class>
 <widget class="QDialog" name="cubedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>正方体体积计算</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-70</x>
     <y>0</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/iocFile/images/2.ico);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="cubeTbWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>171</y>
     <width>521</width>
     <height>231</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>边长</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>体积</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lookButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>75</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>体积：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="volumeEdit">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>75</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculateButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>24</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>24</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>边长：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
